test/labeling/test_convergence.py

```diff
@@ -90,7 +90,7 @@ def test_labeling_convergence(self) -> None:
         self.assertEqual(L_train.shape, (self.N_TRAIN, len(labeling_functions)))
 
         # Train LabelModel
-        label_model = LabelModel(cardinality=self.k, verbose=False)
+        label_model = LabelModel(cardinality=self.cardinality, verbose=False)
```

Comment:
There is a cls.k above that you will also have to change

Reply:
Actually, it is strange that the test passed despite this.

Reply:
it didn't, Travis would have caught it. fixing